java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5803_java.html

```diff
@@ -0,0 +1,30 @@
+<p><code>@VisibleForTesting</code>&nbsp;can be used to mark methods and fields whose visibility restrictions have been relaxed more than necessary for
```

Comment:
`&nbsp;` is suspicious here. Also, RSPEC mentions classes, while here we only have `methods and fields`. I guess you should re-generate the  metadada.